src/api/public/apidocs-new/paths/source_project_name_cmd_addchannels.yaml

```diff
@@ -0,0 +1,16 @@
+post:
+  summary: Add channels to a source
+  description: Add channels to a source
```

Comment:
```suggestion
```
If the description is the same as the summary, it doesn't add value. Better remove it.